core/src/main/java/com/datastrato/gravitino/connector/BaseCatalog.java

```diff
@@ -57,6 +65,9 @@ public abstract class BaseCatalog<T extends BaseCatalog>
   // The object you used is not stable, don't use it unless you know what you are doing.
   @VisibleForTesting public static final String CATALOG_OPERATION_IMPL = "ops-impl";
 
+  // Underlying access control system plugin for this catalog.
+  private volatile BaseAuthorization<?> authorization;
```

Comment:
The concern is that each time when you use this `authorization` you have to check if it is null or not, so it may be not so convenient.

Reply:
I will implement a DefaultAuthorization class in the next PR, It only writes an authorization audit log, doesn't push down authorization in the underlying system.